README.md

```diff
@@ -155,3 +155,25 @@ Otherwise, if you need it to be tested with either the indexer or MusicBrainz Se
 5. Use this tag to [set `MB_SOLR_VERSION` in MusicBrainz Docker Compose project](https://github.com/metabrainz/musicbrainz-docker?tab=readme-ov-file#local-development-of-musicbrainz-solr).
 
 See also [The Debugger’s Guide to MusicBrainz Solr](HACKING.md).
+
+## Helper commands
+
+SolrCloud collections backups from the MusicBrainz Solr cluster
+powering musicbrainz.org are made available twice a week to save
+mirror owners the initial cost of indexing the whole MusicBrainz data
+from scratch which can take hours and significant resources.
+Those backups are provided as Zstandard-compressed tar archives.
+Helper commands to handle these archives are provided in Docker images:
+
+   fetch-backup-archives
+   load-backup-archives
+   remove-backup-archives
+
+In May 2025, the measured time for fetching was 12 min (depending on your bandwidth,)
```

Comment:
Is 12 min correct for what I understand were supposed to be 60 GB? If so, wow, that really does depend on bandwidth, I would probably get at most a quarter of the speed. Should we mention the size again here for clarity?

Reply:
Size reminded in https://github.com/metabrainz/mb-solr/commit/7331f363d7afa3ecfc7d21fa9519644ff54c16f2.